spec/spec_helper.rb

```diff
@@ -29,8 +29,6 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 
-$in_travis = !ENV['TRAVIS'].nil? && ENV['TRAVIS'] == 'true'
```

Comment:
If your skipping tests for travisCI, that should be a flag that something isn't right with the test itself.